### DISCUSSION TITLE
Dịch k-nn, autoencoder, unsupervised pretraining

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#5] [Terminology] [answered]
Em mới bắt gặp vài từ sau, xin ý kiến của mọi người ạ
* `k-nearest neighbor` -> `k điểm lân cận`
* `autoencoder` -> `bộ tự mã hóa` hoặc `mạng tự mã hóa`
* `unsupervised pretraining` -> `tiền huấn luyện không giám sát`

**Comment by lkhphuc** [comment]
* `k điểm lân cận (nhất)`
2 từ kia có trong glossary của D2L rồi thì phải.

**Comment by cuongvng** [comment, marked as answer]
`k- lân cận gần nhất` mn nghĩ sao?

**Reply by lkhphuc** [reply]
`lân cận` gồm nghĩa `gần` rồi mà nhỉ?

**Reply by cuongvng** [reply]
em thấy knn ở VN hay dịch là `k- lân cận gần nhất` hoặc `k- láng giềng gần nhất`.
nếu dịch `neighbor` là `điểm` thì sao không để luôn là `k-điểm gần nhất`?

**Reply by thanhcsf** [reply]
Vậy chỗ này chốt **k-nearest neighbor** là **k-điểm gần nhất** anh em nha.

**Comment by rootonchair** [comment]
`autoencoder` là tên riêng. Lúc trước mình có thống nhất tên riêng mạng không dịch nên em thấy nên giữ nguyên ạ

**Reply by cuongvng** [reply]
mình nghe không giống tên riêng lắm, có nguồn nào chỉ ra không?

**Reply by lkhphuc** [reply]
Mình nghĩ không phải tên riêng. Autoencoder là tên một architecture rồi, giống như GAN, seq2seq. Tên riêng thì phải là Resnet (vẫn dịch trong D2L), Transformer, etc.

**Reply by rootonchair** [reply]
Vậy em nhầm. Đôi khi em thấy từ này được viết hoa
